src/IRrecv.h

```diff
@@ -22,7 +22,7 @@
 // Marks tend to be 100us too long, and spaces 100us too short
 // when received due to sensor lag.
 #define MARK_EXCESS   50U
-#define RAWBUF       100U  // Default length of raw capture buffer
+#define RAWBUF       150U  // Default length of raw capture buffer
```

Comment:
Increasing the size of the buffer should be done when creating the `IRrecv` object, not here.
See `CAPTURE_BUFFER_SIZE` in example code:
https://github.com/markszabo/IRremoteESP8266/blob/master/examples/IRrecvDumpV2/IRrecvDumpV2.ino#L48
&
https://github.com/markszabo/IRremoteESP8266/blob/master/examples/IRrecvDumpV2/IRrecvDumpV2.ino#L102
for how to do that.